src/current/advisories/a110363.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Technical Advisory 110363
+advisory: A-110363
+summary: Clusters using row-level TTL and upgraded from v22.2.x to 23.1.9 may see nodes crashing due to a necessary metadata field being erroneously deleted.
+toc: true
+affected_versions: v22.2.x
```

Comment:
Would we not want to list v23.1.9 here as well? Or even have "Clusters running v22.2.x upgrading to v23.1.9"?
It seems like there are some other cases of a sentence here instead of just the version numbers, if you check out this page: https://deploy-preview-17872--cockroachdb-docs.netlify.app/docs/advisories/

Reply:
I considered adding v23.1.9 but wasn't sure about that. I like your suggestion of adding a complete sentence -- seems clearer to me. @rimadeodhar does that sound okay?

Reply:
SGTM.

Reply:
@dikshant we already say that in the summary. Not sure we need to repeat it for affected versions 🤔

Reply:
Okay sounds good.